kpi/deployment_backends/kobocat_backend.py

```diff
@@ -1286,9 +1286,19 @@ def transfer_submissions_ownership(
 
     def transfer_counters_ownership(self, new_owner: 'auth.User'):
 
+        # Addon usage is not relevant to new owner, so we zero it
         NLPUsageCounter.objects.filter(
             asset=self.asset, user=self.asset.owner
-        ).update(user=new_owner)
+        ).update(
+            user=new_owner,
+            counters=UpdateJSONFieldAttributes(
```

Comment:
Ideally, I think it would be better to use a custom function that allows for removing these keys, rather than changing them to 0. But psql functions are still new territory for me, so it might take me a bit to figure out how. Let me know if you think it would be worth it.